moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -155,8 +155,6 @@ class RobotTrajectory
    */
   double getWayPointDurationFromStart(std::size_t index) const;
 
-  [[deprecated]] double getWaypointDurationFromStart(std::size_t index) const;
```

Comment:
Still need to delete the definition of this function from `robot_trajectory.cpp`